packages/eslint-plugin/docs/rules/no-throw-literal.mdx

```diff
@@ -12,104 +12,14 @@ import TabItem from '@theme/TabItem';
 It is considered good practice to only `throw` the `Error` object itself or an object using the `Error` object as base objects for user-defined exceptions.
 The fundamental benefit of `Error` objects is that they automatically keep track of where they were built and originated.
 
-This rule restricts what can be thrown as an exception. When it was first created, it only prevented literals from being thrown (hence the name), but it has now been expanded to only allow expressions which have a possibility of being an `Error` object. With the `allowThrowingAny` and `allowThrowingUnknown`, it can be configured to only allow throwing values which are guaranteed to be an instance of `Error`.
+This rule restricts what can be thrown as an exception.
 
-## Examples
+:::warn
```

Comment:
```suggestion
:::warning
```

It looks like there is no `warn` admonition in Docusaurus :smile: , it should be `warning` or `caution`

https://deploy-preview-8701--typescript-eslint.netlify.app/rules/no-throw-literal

Reply:
D'oh, thanks!